include/staff/department.inc.php

```diff
@@ -115,6 +115,19 @@
                 </span>
             </td>
         </tr>
+
+        <tr>
+            <td><?php echo __('Ticket Unassignment'); ?>:</td>
+            <td>
+                <span>
+                <input type="checkbox" name="unassign_tickets_dept" <?php echo
+                $info['unassign_tickets_dept']?'checked="checked"':''; ?>>
+                <?php echo __('Unassign ticket assignment on department transfer'); ?>
```

Comment:
Maybe the term should be something like "Unassign ticket when transferring to another department" or just "Unassign ticket on,department transfer".